fleetshard/pkg/util/hash.go

```diff
@@ -0,0 +1,18 @@
+package util
+
+import (
+	"crypto/md5"
+	"encoding/json"
+)
+
+func MD5SumFromJSONStruct(in interface{}) ([16]byte, error) {
+	var sum [16]byte
+
+	b, err := json.Marshal(in)
+	if err != nil {
+		return sum, err
+	}
+
+	sum = md5.Sum(b)
+	return sum, nil
```

Comment:
nit: you can omit the temporary variable.

```suggestion
	b, err := json.Marshal(in)
	if err != nil {
		return [16]byte{}, err
	}
	return md5.Sum(b), nil
```

Reply:
Done